### PR TITLE
refactor(WebGL): convert HTML tables to Markdown

### PR DESCRIPTION
**Pull request by caugner**
### Description

Converts verbose HTML tables to Markdown.

### Motivation

This page is causing trouble during machine translation due to its length, and converting the tables should help.

### CODE REVIEW

File: files/en-us/web/api/webgl_api/constants/index.md [hunk 1]
```diff
@@ -53,744 +53,131 @@ Constants passed to {{domxref("WebGLRenderingContext.clear()")}} to clear buffer
...
-        <code>DECR_WRAP</code>.
-      </td>
-    </tr>
-    <tr>
-      <td><code>STENCIL_PASS_DEPTH_PASS</code></td>
-      <td>0x0B96</td>
-      <td>
-        Passed to <code>getParameter</code> to get the current stencil fail
-        function should the depth buffer test pass. Should return KEEP, REPLACE,
-        INCR, DECR, INVERT, INCR_WRAP, or DECR_WRAP.
```
Comment (caugner) [inline files/en-us/web/api/webgl_api/constants/index.md:516]: Note: The keywords `KEEP`, `REPLACE` etc are not in backticks, but should be.

**Comment by caugner** [comment]
Here's a diff, comparing the pretty-printed `innerHTML` of the rendered `article` element, between [production](https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API/Constants) and the [PR preview](https://pr36073.content.dev.mdn.mozit.cloud/en-US/docs/Web/API/WebGL_API/Constants):

https://www.diffchecker.com/S6Di7N33/

File: files/en-us/web/api/webgl_api/constants/index.md [hunk 2]
```diff
@@ -811,137 +198,27 @@ Constants passed to {{domxref("WebGLRenderingContext.getVertexAttrib()")}}.
...
-      </td>
-    </tr>
-    <tr>
-      <td><code>POLYGON_OFFSET_FILL</code></td>
-      <td>0x8037</td>
-      <td>
-        Passed to <code>enable</code>/<code>disable</code> to turn on/off the
-        polygon offset. Useful for rendering hidden-line images, decals, and or
-        solids with highlighted edges. Can also be used with
-        <code>getParameter</code> to query the scissor test.
```
Comment (caugner) [inline files/en-us/web/api/webgl_api/constants/index.md:906]: Note: This contains "Useful for rendering hidden-line images, decals, **and or** solids with highlighted edges." and "and or" was replaced by "and" here.

File: files/en-us/web/api/webgl_api/constants/index.md [hunk 3]
```diff
@@ -53,744 +53,131 @@ Constants passed to {{domxref("WebGLRenderingContext.clear()")}} to clear buffer
...
-    </tr>
-  </tbody>
-</table>
+| Constant name                      | Value  | Description                                                                                                                                                                                                                                  |
+| ---------------------------------- | ------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `BLEND_EQUATION`                   | 0x8009 | Passed to `getParameter` to get the current RGB blend function.                                                                                                                                                                              |
+| `BLEND_EQUATION_RGB`               | 0x8009 | Passed to `getParameter` to get the current RGB blend function. Same as BLEND_EQUATION                                                                                                                                                       |
```
Comment (bsmth) [inline files/en-us/web/api/webgl_api/constants/index.md:105]:
```suggestion
| `BLEND_EQUATION_RGB`               | 0x8009 | Passed to `getParameter` to get the current RGB blend function. Same as `BLEND_EQUATION`                                                                                                                                                       |
```

File: files/en-us/web/api/webgl_api/constants/index.md [hunk 4]
```diff
@@ -1040,231 +289,42 @@ Constants passed to {{domxref("WebGLRenderingContext.hint()")}}
...
+| `MAX_VARYING_VECTORS`              | 0x8DFC |                                                                                                                                                                                                  |
+| `MAX_COMBINED_TEXTURE_IMAGE_UNITS` | 0x8B4D |                                                                                                                                                                                                  |
+| `MAX_VERTEX_TEXTURE_IMAGE_UNITS`   | 0x8B4C |                                                                                                                                                                                                  |
+| `MAX_TEXTURE_IMAGE_UNITS`          | 0x8872 | Implementation dependent number of maximum texture units. At least 8.                                                                                                                            |
```
Comment (bsmth) [inline files/en-us/web/api/webgl_api/constants/index.md:308]:
```suggestion
| `MAX_TEXTURE_IMAGE_UNITS`          | 0x8872 | Implementation-dependent number of maximum texture units. At least 8.                                                                                                                            |
```

File: files/en-us/web/api/webgl_api/constants/index.md [hunk 5]
```diff
@@ -53,744 +53,131 @@ Constants passed to {{domxref("WebGLRenderingContext.clear()")}} to clear buffer
...
-      </td>
-    </tr>
-  </tbody>
-</table>
+| Constant name              | Value  | Description                                                                                                                                 |
+| -------------------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| `ZERO`                     | 0      | Passed to `blendFunc` or `blendFuncSeparate` to turn off a component.                                                                       |
+| `ONE`                      | 1      | Passed to `blendFunc` or `blendFuncSeparate` to turn on a component.                                                                        |
+| `SRC_COLOR`                | 0x0300 | Passed to `blendFunc` or `blendFuncSeparate` to multiply a component by the source elements color.                                          |
+| `ONE_MINUS_SRC_COLOR`      | 0x0301 | Passed to `blendFunc` or `blendFuncSeparate` to multiply a component by one minus the source elements color.                                |
```
Comment (bsmth) [inline files/en-us/web/api/webgl_api/constants/index.md:75]:
```suggestion
| `SRC_COLOR`                | 0x0300 | Passed to `blendFunc` or `blendFuncSeparate` to multiply a component by the source element's color.                                          |
| `ONE_MINUS_SRC_COLOR`      | 0x0301 | Passed to `blendFunc` or `blendFuncSeparate` to multiply a component by one minus the source element's color.                                |
```

**Review by bsmth — APPROVED** [review]
LGTM with some minor suggestions, tnx 👍🏻

**Pull request merged**